README.md

```diff
@@ -63,7 +63,7 @@ _Long-form documentation for this section can be found [here](docs/build_and_loa
 ./phg create-maf-vcf --db-path /path/to/dbs --bed /my/bed/file.bed --reference-file /my/ref.fasta --maf-dir /my/maf/files -o /path/to/vcfs
 
 ## Convert GVCF to HVCF: use this if you have GVCF files created by PHG, but do not have MAF or h.vcf files
```

Comment:
This header should say Optional or instead of using create-maf-vcf.

Reply:
changed.